build.xml

```diff
@@ -180,6 +182,36 @@
         <phingcall target="eslint-fix"/>
     </target>
 
+    <target name="npm" description="Installs node modules.">
+        <exec executable="${path.npm.executable}" dir="${path.root}" logoutput="true" passthru="true" checkreturn="true">
+            <arg value="prune"/>
+            <arg value="--no-audit"/>
+        </exec>
+
+        <exec executable="${path.npm.executable}" dir="${path.root}" logoutput="true" passthru="true" checkreturn="true">
+            <arg value="install"/>
+        </exec>
+
```

Comment:
You can call the parent NPM target, in that case, you don't need to change anything in a project if we make a change in this target in framework's build.xml file. (for example, remove prune :D)

Reply:
Thx,I changed it